frontend/packages/pipelines-plugin/src/components/pipelineruns/logs/MultiStreamLogs.tsx

```diff
@@ -145,7 +145,8 @@ export const MultiStreamLogs: React.FC<MultiStreamLogsProps> = ({
       >
         <div className="odc-multi-stream-logs__container__logs" ref={scrollPane}>
           {containers.map((container, idx) => {
-            const resourceStatus = containerToLogSourceStatus(containerStatus[idx]);
```

Comment:
The order of object in both container and containerStatus is not guaranteed, so we can't relay on container `idx` to get the correct status.